packages/components/src/notice/index.tsx

```diff
@@ -52,6 +52,20 @@ function getDefaultPoliteness( status: NoticeProps[ 'status' ] ) {
 	}
 }
 
+function getStatusLabel( status: NoticeProps[ 'status' ] ) {
+	switch ( status ) {
+		case 'warning':
+			return __( 'Warning notice' );
+		case 'info':
+			return __( 'Information notice' );
+		case 'error':
+			return __( 'Error notice' );
+		case 'success':
+		default:
```

Comment:
Should we remove the `case 'success':` line here? The `default` already acts as a fallback, so that line doesn't serve a purpose 🤔 
```suggestion
		default:
```

Reply:
I guss that for consistency the redundant case should be removed from `getDefaultPoliteness` as well.